main_test.go

```diff
@@ -178,14 +211,20 @@ var _ = Describe("authentic", func() {
 
 			It("returns 401 response in Gin middleware", func() {
 				mockContext.Request.Header["Authorization"] = []string{"Bearer " + badToken}
-				authentic.NewMiddlewareCreator().CreateGinMiddleware()(mockContext)
+				middlewareCreator.CreateGinMiddleware()(mockContext)
 				json.NewDecoder(rec.Body).Decode(&body)
 				Expect(rec.Code).To(Equal(401))
 				Expect(body.Message).To(Equal("Unauthorized"))
 			})
 
+			It("returns 403 response in Gin middleware due to expired token", func() {
```

Comment:
403 => 401 in description?

Reply:
Ah yea good catch e88b57937c832a793e0ee5307b9801af9d0cae90